deployments/helm-chart/crds/appprotectdos.f5.com_apdoslogconfs.yaml

```diff
@@ -0,0 +1,70 @@
+apiVersion: apiextensions.k8s.io/v1
+kind: CustomResourceDefinition
+metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.4.0
+  creationTimestamp: null
+  name: apdoslogconfs.appprotectdos.f5.com
+spec:
+  group: appprotectdos.f5.com
+  names:
+    kind: APDosLogConf
+    listKind: APDosLogConfList
+    plural: apdoslogconfs
+    singular: apdoslogconf
+  preserveUnknownFields: false
+  scope: Namespaced
+  versions:
+    - name: v1beta1
```

Comment:
is this v1 now with k8s v1.22 ? https://kubernetes.io/docs/reference/kubernetes-api/extend-resources/custom-resource-definition-v1/